make/tools.mk

```diff
@@ -100,6 +100,21 @@ endif
 # Build openocd without FTDI (yes | no)
 OPENOCD_FTDI ?= yes
 
+
+# $1 URL
```

Comment:
I should use this for all the other downloads in tools.mk but I'm feeling lazy atm :stuck_out_tongue: . The only download if newer part is important for me installing Qt (long story... Qt picks a bad mirror by default..) so I needed to make it work on Windows and decided it would be useful for more than just Qt downloads hence the template.